deps/Chakra.ICU/Chakra.ICU.Build.props

```diff
@@ -20,6 +20,11 @@
         %(PreprocessorDefinitions)
       </PreprocessorDefinitions>
 
+      <PreprocessorDefinitions>
+        U_DISABLE_RENAMING=1;
+        %(PreprocessorDefinitions)
+      </PreprocessorDefinitions>
+
       <!-- Default ICU Configuration (see source\common\common.vcxproj) -->
```

Comment:
Do you actually build using the "common.vcxproj" file, or do you build ICU using some other mechanism?
(Sorry -- I'm a complete noob when it comes to how ChakraCore is built...)

Reply:
We avoided the default vcxprojs for a few reasons:
* They enforce building DLLs, which I wasn't a huge fan of
* In general, they don't allow any options to be overridden -- for instance, the warning output is exceptionally noisy, but its impossible to disable warnings because it unanimously sets `<WarningLevel>Level3</WarningLevel>`, for example. Another concerning option that it set like this was turning off ASLR and DYNAMICBASE, which just seems really unnecessary if not dangerous
* They hardcoded where binaries go, which didn't sit well with the rest of our vcxprojs (there is a custom-build hierarchy of variables and such that make all of the subprojects use a common IntDir and OutDir.

Basically, instead of building <icu>/source/common.vcxproj, we build Chakra.ICU.Common.vcxproj -- theyre functionally equivalent. I took a lot of the inspiration for this from the gyp-based custom build system that Steven Loomis implemented for Node.